docs/api/security/lifecycle/psa_lifecycle.dot

```diff
@@ -0,0 +1,11 @@
+digraph {
```

Comment:
Query: What is this file for? Should it be here?

Reply:
used this file in DOT language for describing the lifecycle state machine.
please suggest the best place for it.